### PR TITLE
Update away image and image paths

### PR DESCRIPTION
**Pull request by alexiamandeville**
Updated PNG
Updated image paths to point to the file in the program rather than a remote source

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8078/app_12660_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8078/HighFidelity-Beta-PR12660-74c654690f1a0b950697779ed1c41de1522b7bdc.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8078/HighFidelity-Beta-PR12660-74c654690f1a0b950697779ed1c41de1522b7bdc.exe

File: scripts/system/away.js [hunk 1]
```diff
@@ -37,7 +38,7 @@ var OVERLAY_DATA_HMD = {
     localRotation: {x: 0, y: 0, z: 0, w: 1},
     width: OVERLAY_WIDTH,
     height: OVERLAY_HEIGHT,
-    url: "http://hifi-content.s3.amazonaws.com/alan/production/images/images/Overlay-Viz-blank.png",
+	url: Script.resolvePath("assets/images/Overlay-Viz-blank.png"),
```
Comment (sethalves) [inline scripts/system/away.js]: the indentation for the changes in this file seem wrong.  it should always be multiples of 4 characers

**Comment by alexiamandeville** [comment]
Test plan: 
1. Run interface without VR
2. Press ESC to set Away
3. Make sure image looks correct
4. Enter VR mode
5. Make sure image looks correct

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8094/app_12660_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8094/HighFidelity-Beta-PR12660-921b19139d278398ea8cd7bbb6b934f6db15ac4a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8094/HighFidelity-Beta-PR12660-921b19139d278398ea8cd7bbb6b934f6db15ac4a.exe

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8284/app_12660_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8284/HighFidelity-Beta-PR12660-921b19139d278398ea8cd7bbb6b934f6db15ac4a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8284/HighFidelity-Beta-PR12660-921b19139d278398ea8cd7bbb6b934f6db15ac4a.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
New away image appears and looks better.

**Pull request merged**